GridTariffAPI.lib/Controllers/v1/TariffQueryController.cs

```diff
@@ -16,7 +16,7 @@ namespace GridTariffApi.Lib.Controllers.v1
     [ApiController]
     [Authorize]
     [ApiVersion("1.0")]
-    [Route("api/{v:apiVersion}/powerbased")]
+    [Route("digin/api/{v:apiVersion}")]
```

Comment:
Controlleren ligger i Lib-prosjektet, men tenker det er helt OK om de som vil bruke controllerne våre får /digin/ som del av url'en. Vi kan evt. gjøre noe med det hvis det blir et tema.

Reply:
Godt poeng. Ja, ser ingen problemer med digin-prefix, det er jo bare noe man tar høyde for i digin-listen over endepunkt for nettselskaper. Are var vel på et tidspunkt også åpen for at det kunne bli del av standarden.